source/botan/utils/http_util/http_util.d

```diff
@@ -249,18 +247,18 @@ string httpTransact(in string hostname, in string message)
     } else {*/
         Socket socket = new TcpSocket(new InternetAddress(hostname, 80));
         scope(exit) socket.close();
-        SocketStream stream = new SocketStream(socket);
-        stream.writeString(message);
+        socket.send(message);
 
         Appender!string in_buf;
         // Skip HTTP header.
         while (true)
         {
-            auto line = stream.readLine();
-            if (!line.length)
+            char[1024] buffer;
```

Comment:
I'm not 100% sure this is the right way to handle this. It is slightly different than the original code, in that it read the entire stream rather than only reading the headers.